fastembed/text/onnx_models.py

```diff
@@ -0,0 +1,133 @@
+supported_flag_models = [
```

Comment:
I'd prefer to have a flat models.json or similar config-like file, instead of 3 different lists. 

If we want to separate the models into different classes and files, we should move this completely to those corresponding files — that way the places to update in any new model addition are:

1. New class in a file
2. Embedding Registry
3. Tests

That's it

Reply:
I am ok with moving lists to the corresponding implementation. But having a single list doesn't work. It creates very ugly hacks as with the "exclude" param of the list models method.